context/app/static/js/components/Detail/Visualization/style.js

```diff
@@ -8,14 +8,7 @@ const headerFixedHeight = 64;
 const vitessceFixedHeight = 600;
 
 const StyledHeader = styled.div`
-  margin-bottom: 6px;
-  display: flex;
-`;
-
-const StyledHeaderLeft = styled.div`
-  margin-bottom: 6px;
-  display: flex;
-  flex-direction: column;
+  margin-bottom: 12px;
```

Comment:
```suggestion
  margin-bottom: ${(props) => props.theme.spacing(1.5)}px;
```
I used 6px elsewhere, but this will get cleaned up this week when we review the UI details

Reply:
The value should also come from spacing in the theme, but this was from before I was doing so.

Reply:
I added this in.  It can be changed later, but the buttons looked a little close for my taste.  Something to bring up once spacing gets a review.